WINDOWS_BASELINE.md

```
@@ -0,0 +1,142 @@
+  tests/windows/windows-user-principal.test.ts `
+  tests/codex-integration/codex-user-identity.test.ts `
+  tests/codex-integration/codex-inject-integration.test.ts
+```
+
+期望：≥ 74 pass / ≤ 2 flaky fail（单跑必然复绿） / 1 skip。
+
+## 全量套件整体基线（历史快照数字）
+
+`bun run test` 在这台 Windows 机器上的全量结果（参考 fork 快照 `d881140`）：
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)